core/dbt/tests/fixtures/project.py

```diff
@@ -598,7 +599,7 @@ def project_setup(
 # to pull in the other fixtures individually to access their information.
 @pytest.fixture(scope="class")
 def project(
-    project_files,
     project_setup: TestProjInfo,
+    project_files,
```

Comment:
I'm not asking for a change here, but I would like to understand the context. Does the ordering of the fixture arguments here mean something? Is there a reason the `project_setup` fixture has to be come before `project_files`? This is the kind of thing I would love to see in a commit comment so that I can better understand why something is changing while reviewing.

Reply:
Yes, adding a comment in the following commit.
The order of the argument determines the order of fixtures being executed(if they need to).
So we do all setups, then write the project files. Setup should be independent of project file write.

Another reason is for the happy path fixture, we do not use the `dbt_project.yml` fixture, instead, we want to use the files in the fixture folder, changing the order makes sure that the tests run with the intended files